pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -156,7 +157,11 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 
 	// provide any config overrides
 	cfg.NodeName = options.NodeName
-	cfg.StreamingConnectionIdleTimeout = 5 * time.Minute // TODO: should be set
+	if server.StreamingConnectionIdleTimeout == 0 {
+		cfg.StreamingConnectionIdleTimeout = 5 * time.Minute
```

Comment:
Add the change to godeps in a separate commit with the upstream pull request number in the commit title. See other commits in master for examples